R/save_to_gama.R

```diff
@@ -50,17 +52,17 @@ generate_obsrate <- function(obsrate, names){
 #' @rdname save_to_gama
 #' @export
 
-save_to_gama <- function(exp, parameter_xml_file) UseMethod("save_to_gama")
+save_to_gama <- function(exp, filename = NULL, path = NULL) UseMethod("save_to_gama")
```

Comment:
There is a `pmap()` call in this function that should be replaced by an `mapply()` call.

Reply:
done